### DISCUSSION TITLE
About improving korean l10n

### DISCUSSION THREAD

**Discussion by HotoRas** [community/community#118873] [Mobile] [answered]
### Select Topic Area

Product Feedback

### Body

It's already thankful to service GitHub Mobile into korean, my primary language.
However, the l10n is somewhat misleading- like it's machine translated.
Maybe it'd be great to open a survay about improving it, so the newcomers won't mislead due to the localized terms.

**Comment by taki-on** [comment]
Hey! I really appreciate your report! I totally agree with you that the Korean translations sometimes sound like machine-translation, and there could be misleading translations. Would you mind giving us some examples you've found? Like the badest ones you think. I will ping our l10n team and set those as higher prioritized. Thank you so much again and I am looking forwards to hearing from you back!

**Reply by HotoRas** [reply]
All I thought it for misleading got reverted- maybe being reworked from now. However, like this screenshot..
![Set theme as Light, Dark, or Follow system. Language set to Korean(한국어).](https://github.com/community/community/assets/34373595/062950fd-72fd-4e88-9702-4b0c562287f5)
`Follow system` is more flawless if it's localized to `시스템 설정에 따름`, since the one now doesn't indicate object- we all know it's `settings` but- that appears in the middle of the sentense in the case of Korean.

**Reply by taki-on** [reply, marked as answer]
Thank you for sharing the screenshots! I will go through Korean translations and will ping our l10n team to improve them including your suggestion! Please keep reporting if you find any odd translations. We really appreciate it! 🙇 감사합니다!
